### PR TITLE
Extend Okapi Console with ability to undeploy modules

### PR DESCRIPTION
**Pull request by MikeTaylor**
In the Modules section of the Okapi Console, the popup that you get when clicking on a module name includes a section listing all deployments, and for each one a button to undeploy.

The UX is horrible, but it gets the job done ... and this is ui-developer.

### CODE REVIEW

**Comment by MikeTaylor** [comment]
@wafschneider I did this for a very pragmatic reason: I am deploying _a lot_ of instances of modules on my local box as I test `add-app-to-platform`. I hope it's of use to others, too.

File: src/settings/OkapiConsole/Modules.js [hunk 1]
```diff
@@ -33,7 +33,88 @@ function maybeRenderInterfaces(detail, tag, isPermissions) {
   );
 }
 
-function ModuleDetail({ detail }) {
+
+function undeploy(okapiKy, callout, id, instId) {
+  return okapiKy.delete(`_/discovery/modules/${id}/${instId}`)
```
Comment (wafschneider) [inline src/settings/OkapiConsole/Modules.js:38]: Just so you are aware -- Okapi will allow you to do this even if the module is in use by a tenant, so you can easily break an environment this way.

**Comment by MikeTaylor** [comment]
Thanks, @wafschneider. I will come back later and add the Are You Sure? barrier, but note that the button doesn't even appear if the user doesn't have the `okapi.discovery.delete` permission -- which no-one does in a seriously environment, right?

**Comment by MikeTaylor** [comment]
@wafschneider I added the confirmation modal: please review https://github.com/folio-org/ui-developer/pull/344

**Pull request merged**